auth-api/src/auth_api/services/org.py

```diff
@@ -149,11 +149,12 @@ def create_org(org_info: dict, user_id):
 
         ProductService.create_subscription_from_bcol_profile(org.id, bcol_profile_flags)
 
-        Org._create_payment_for_org(mailing_address, org, payment_info, True)
+        payment_account_status, error = Org._create_payment_for_org(mailing_address, org, payment_info, True)
 
-        # TODO do we have to check anything like this below?
-        # if payment_account_status == PaymentAccountStatus.FAILED:
-        # raise BusinessException(Error.ACCOUNT_CREATION_FAILED_IN_PAY, None)
+        if payment_account_status == PaymentAccountStatus.FAILED and error is not None:
+            logger.error(f"Account update payment Error: {error}")
```

Comment:
so if this fails, it doesn't return a 500 error? Hmm that's not so great

Reply:
I changed to warning, should be fine